Gems/Camera/Code/Source/CameraComponentController.cpp

```diff
@@ -58,7 +58,7 @@ namespace Camera
                         ->Attribute(AZ::Edit::Attributes::Min, MinFoV)
                         ->Attribute(AZ::Edit::Attributes::Suffix, " degrees")
                         ->Attribute(AZ::Edit::Attributes::Step, 1.f)
-                        ->Attribute(AZ::Edit::Attributes::Max, AZ::RadToDeg(AZ::Constants::Pi) - 0.0001f)       //We assert at fovs >= Pi so set the max for this field to be just under that
+                        ->Attribute(AZ::Edit::Attributes::Max, AZ::RadToDeg(AZ::Constants::Pi))
```

Comment:
I just want to check you were able to successfully set the FOV to 180 and things didn't assert (perhaps this was an issue with the legacy renderer and so is no longer an issue). To verify this you will need to ensure asserts are enabled for your build (there's a chance they might not be, at least visible ones).

The best way to verify this is to create a new file at the root of the repo called `user.cfg` and add a single line with

```
sys_asserts = 2
```

This will throw-up a big pop-up if the assert fires that is alluded to in this comment. 

Including a video of verifying this would be great too. If all seems well then this looks good, otherwise we might need to find another fix.

Thanks!

Reply:
Thanks for the advice on verifying the change! After testing again, I do get asserts, and probably reasonably so because when the camera fov is at 180 degrees, the view flips. You can see this behavior in the video below.

Possibly a better way to address issue https://github.com/o3de/o3de/issues/4081 is by updating the tooltip.

Reply:
Found that the assert is due to fov >= 180 impacting the CascadeShadowCameraConfiguration: See [line 51](https://github.com/o3de/o3de/blob/development/Gems/Atom/Feature/Common/Code/Source/CoreLights/DirectionalLightFeatureProcessor.cpp#L51). 

Let me know what you think! I'll probably revert the change and update the tooltip instead.

Reply:
I believe the correct fix here is to update the tooltip, not increase the max FOV to be 180. A field of view of 180 would give you an invalid camera frustum, which is why it is limited to be less than 180.

Reply:
Here is a diagram that shows how the FOV (90 degrees in this case) relates to the camera frustum. If you were to increase that all the way to 180, the far plane would be infinitely large.

https://i.stack.imgur.com/JomoW.png

Reply:
Agree with Tommy here, should just update the tooltip and not allow 180 degree FOV

Reply:
Updating the tool-tip sounds good, thanks for the feedback and details @amzn-tommy and @antonmic 👍